### PR TITLE
added json accesses for annual reports

### PR DESCRIPTION
**Pull request by VSBalanchuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Liubov-H — APPROVED** [review]

File: EPlast/EPlast.BLL/Interfaces/AnnualReport/IAnnualReportAccessService.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using EPlast.BLL.DTO.AnnualReport;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EPlast.DataAccess.Entities;
```
Comment (dashapenkova) [inline EPlast/EPlast.BLL/Interfaces/AnnualReport/IAnnualReportAccessService.cs:4]: Do you need all these usings?

File: EPlast/EPlast.BLL/Services/AnnualReport/AnnualReportAccessService.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using EPlast.BLL.Services.Interfaces;
+using EPlast.DataAccess.Entities;
+using EPlast.DataAccess.Repositories;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Internal;
+using Newtonsoft.Json;
```
Comment (dashapenkova) [inline EPlast/EPlast.BLL/Services/AnnualReport/AnnualReportAccessService.cs:8]: Same. Delete and sort usings

**Review by dashapenkova — APPROVED** [review]

File: EPlast/EPlast.BLL/Interfaces/AnnualReport/IAnnualReportAccessService.cs (hunk 1 above)
Comment (Bodasyuk) [reply EPlast/EPlast.BLL/Interfaces/AnnualReport/IAnnualReportAccessService.cs]: Sort usings

File: EPlast/EPlast.BLL/Services/UserAccess/UserAccessService.cs [hunk 3]
```diff
@@ -2,10 +2,11 @@
 using EPlast.BLL.Interfaces.City;
 using EPlast.BLL.Interfaces.Club;
 using EPlast.BLL.Interfaces.UserAccess;
+using EPlast.BLL.Interfaces.Region;
 using EPlast.DataAccess.Entities;
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using EPlast.BLL.Interfaces.Region;
+using EPlast.BLL.Services.Interfaces;
```
Comment (Bodasyuk) [inline EPlast/EPlast.BLL/Services/UserAccess/UserAccessService.cs:9]: Sort usings

File: EPlast/EPlast.BLL/Services/AnnualReport/AnnualReportAccessService.cs (hunk 2 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.BLL/Services/AnnualReport/AnnualReportAccessService.cs]: Thanks, changed it

File: EPlast/EPlast.BLL/Interfaces/AnnualReport/IAnnualReportAccessService.cs (hunk 1 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.BLL/Interfaces/AnnualReport/IAnnualReportAccessService.cs]: Thanks, changed it

File: EPlast/EPlast.BLL/Services/AnnualReport/AnnualReportAccessService.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
+﻿using EPlast.BLL.Services.Interfaces;
+using EPlast.DataAccess.Entities;
+using EPlast.DataAccess.Repositories;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Internal;
```
Comment (Bodasyuk) [inline EPlast/EPlast.BLL/Services/AnnualReport/AnnualReportAccessService.cs]: Sort usings

File: EPlast/EPlast.Tests/Services/UserAcccess/UserAccessServiceTests.cs [hunk 5]
```diff
@@ -8,6 +8,7 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using EPlast.BLL.Interfaces.Region;
+using EPlast.BLL.Services.Interfaces;
```
Comment (Bodasyuk) [inline EPlast/EPlast.Tests/Services/UserAcccess/UserAccessServiceTests.cs:11]: Sort usings

**Review by Bodasyuk — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/UserAccess/UserAccessService.cs (hunk 3 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.BLL/Services/UserAccess/UserAccessService.cs]: Changed

File: EPlast/EPlast.Tests/Services/UserAcccess/UserAccessServiceTests.cs (hunk 5 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.Tests/Services/UserAcccess/UserAccessServiceTests.cs]: Changed it

File: EPlast/EPlast.BLL/Services/AnnualReport/AnnualReportAccessService.cs (hunk 4 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.BLL/Services/AnnualReport/AnnualReportAccessService.cs]: Changed

**Pull request merged**